README.md

```diff
@@ -330,3 +330,19 @@ var objectType = new minim.ObjectType()
   .set('email', 'john@example.com')
   .set('id', 4)
 ```
+
+### Errors
+
+Minim has its own error type that is passed down through the call chain. This allows clients to write chains that may break but do not throw JavaScript errors.
+
+```javascript
+var objectExample = {
+  foo: {
+    bar: 'value 1'
+  }
+};
+
+var objectType = minim.ObjectType(objectExample);
+var value1 = objectType.get('foo').get('bar'); // value of foo.bar
+var notFound = objectType.get('baz').get('foo') // Returns error object
```

Comment:
So this really needs a couple of things (beside a method missing) to work well. Right now, the instance that failed actually gets passed into the error instance, so that an error instance knows what element failed and why it failed. 

But what would really help is if we kept track of where things were being called, so that an error could provide the path that failed.

So to sum it up, it does have some idea where the error happened, but it's not perfect.